Documentation/ceph-upgrade.md

```diff
@@ -89,6 +89,40 @@ you should change to match where your images are located.
         value: "quay.io/k8scsi/csi-resizer:v0.4.0"
 ```
 
+### Retroactively enabling volume expansion
+
+When upgrading to the CSI 2.0 driver with existing persistent volumes, the following considerations should be taken into account when choosing to leverage the volume expansion feature:
+
+#### Remediate existing storage classes
+
+It is also necessary to ensure that the controlling StorageClass have the required changes
+to support Volume Expansion see the documentation about configuring [dynamically expanding volumes](https://rook.io/docs/rook/master/ceph-csi-drivers.html#dynamically-expand-volume).
+
+#### Remediate existing persistent volumes
+
+For any existing persistent volumes that you wish to 'upgrade' to allow expansion, you must
+patch them with some needed metadata.
+
+You can do this by:
+
+1. create a patch file, e.g. `csi-expansion-patch.yaml` with the following:
+
+```yaml
+spec:
+  csi:
+    controllerExpandSecretRef:
+      name: rook-ceph-csi
```

Comment:
is this name correct? can you please double-check with https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/csi/rbd/storageclass.yaml#L39

Reply:
It is not correct _now_.  It was [renamed in this commit](https://github.com/rook/rook/commit/fcc2e21fafc338b55c4b3d6056e50ef900d38fa4) 4 months ago so I believe we should take care to assume what the name would be for existing ceph users.  

How do you think we should handle this part of the instructions?  

* Have the user examine whatever 'version' of the StorageClass they are using to extract the current naming standard that they are using (which changed 4 months back)
* Remediate their existing storageclasses to match the new naming convention
* Recreate the storageclasses with the changes from 4 months back?